static/css/main.css

```diff
@@ -31,3 +31,21 @@ ul {
   margin: 0;
   padding-left: 1em;
 }
+
+@media only screen and (max-width : 450px) {
+  .org {
+    flex: 100%;
```

Comment:
Should be `flex-basis` or `flex: 0 0 <size>` :P
Setting it like this would be like doing `flex: 1 1 100%` which could make the column expand.

Although it does not affect the visual now, it might do in the future.
Change all the occurence of `flex: ..` in all breakpoints.